### PR TITLE
Update nl.json

### PR DESCRIPTION
**Pull request by marvinlinthorst**

### CODE REVIEW

File: locales/nl/nl.json [hunk 1]
```diff
@@ -270,8 +270,7 @@
...
     "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "Is het nodig om specifieke contact- of BTW gegevens toe te voegen aan je bonnen, zoals je volledige bedrijfsnaam, BTW-nummer, of adress, dan kan je dat hier toevoegen.",
-    "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:",
-    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Als je problemen hebt met de \":actionText\" knop, kopieer en plak de URL hieronder\nin je webbrowser:",
```
Comment (caouecs) [inline locales/nl/nl.json:274]: don't delete this line, please

**Review by caouecs — CHANGES_REQUESTED** [review]

**Comment by marvinlinthorst** [comment]
Fixed

File: locales/nl/nl.json [hunk 2]
```diff
@@ -266,8 +266,8 @@
     "If you did not request a password reset, no further action is required.": "Als je geen wachtwoordherstel hebt aangevraagd, hoef je verder niets te doen.",
     "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Als je nog geen account hebt, kan je er een aanmaken door op onderstaande knop te klikken. Na het aanmaken van je account kan je op de uitnodiging in deze mail klikken om die te accepteren:",
     "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "Is het nodig om specifieke contact- of BTW gegevens toe te voegen aan je bonnen, zoals je volledige bedrijfsnaam, BTW-nummer, of adress, dan kan je dat hier toevoegen.",
-    "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:",
```
Comment (caouecs) [inline locales/nl/nl.json:269]: don't delete this line. the difference with the next line is just "you're" and "you’re"

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/nl/nl.json (hunk 2 above)
Comment (marvinlinthorst) [reply locales/nl/nl.json:269]: They are both present:
image

**Comment by caouecs** [comment]
Thank you

**Pull request merged**